src/stan/model/indexing/rvalue.hpp

```diff
@@ -92,58 +70,53 @@ inline T rvalue(const Eigen::Matrix<T, 1, Eigen::Dynamic>& rv,
  * @param[in] depth Depth of indexing dimension.
  * @return Result of indexing vector.
  */
-template <typename T, typename I>
-inline typename boost::disable_if<boost::is_same<I, index_uni>,
-                                  Eigen::Matrix<T, Eigen::Dynamic, 1> >::type
-rvalue(const Eigen::Matrix<T, Eigen::Dynamic, 1>& v,
-       const cons_index_list<I, nil_index_list>& idx, const char* name = "ANON",
-       int depth = 0) {
+template <typename EigVec, typename I,
```

Comment:
Doxygen should include that this also works with row vectors.

Reply:
Gonna go through and update all the docs